docs/quick_start/installation.rst

```diff
@@ -277,14 +278,37 @@ Here are the most common OCIO-specific CMake options (the default values are sho
 - ``-DOCIO_USE_OIIO_FOR_APPS=OFF`` (Set ON to build tools with OpenImageIO rather than OpenEXR)
 - ``-DOCIO_BUILD_PYTHON=ON`` (Set to OFF to not build the Python binding)
 - ``-DOCIO_BUILD_OPENFX=OFF`` (Set to ON to build the OpenFX plug-ins)
-- ``-DOCIO_USE_SSE=ON`` (Set to OFF to turn off SSE CPU performance optimizations)
+- ``-DOCIO_USE_SIMD=ON`` (Set to OFF to turn off SIMD CPU performance optimizations, such as SSE and NEON)
+- ``-DOCIO_USE_SSE2=ON`` (Set to OFF to turn off SSE2 CPU performance optimizations)
+- ``-DOCIO_USE_SSE3=ON`` (Set to OFF to turn off SSE3 CPU performance optimizations)
+- ``-DOCIO_USE_SSSE2=ON`` (Set to OFF to turn off SSSE3 CPU performance optimizations)
+- ``-DOCIO_USE_SSE4=ON`` (Set to OFF to turn off SSE4 CPU performance optimizations)
+- ``-DOCIO_USE_SSE42=ON`` (Set to OFF to turn off SSE4.2 CPU performance optimizations)
+- ``-DOCIO_USE_AVX=ON`` (Set to OFF to turn off AVX CPU performance optimizations)
+- ``-DOCIO_USE_AVX2=ON`` (Set to OFF to turn off AVX2 CPU performance optimizations)
+- ``-DOCIO_USE_AVX512=ON`` (Set to OFF to turn off AVX512 CPU performance optimizations)
+- ``-DOCIO_USE_F16C=ON`` (Set to OFF to turn off F16C CPU performance optimizations)
+- ``-OCIO_USE_SSE2NEON=ON`` (Apple Only; Set to OFF to turn off the SSE2NEON translation performance optimizations on Apple ARM)
 - ``-DOCIO_BUILD_TESTS=ON`` (Set to OFF to not build the unit tests)
 - ``-DOCIO_BUILD_GPU_TESTS=ON`` (Set to OFF to not build the GPU unit tests)
 - ``-DOCIO_USE_HEADLESS=OFF`` (Set to ON to do headless GPU reendering)
 - ``-DOCIO_WARNING_AS_ERROR=ON`` (Set to OFF to turn off warnings as errors)
 - ``-DOCIO_BUILD_DOCS=OFF`` (Set to ON to build the documentation)
 - ``-DOCIO_BUILD_FROZEN_DOCS=OFF`` (Set to ON to update the Python documentation)
 
+Note that OCIO will turn off any specific SIMD CPU performance optimizations if they are not supported 
+by the build target architecture.
+
+On the MacOS, the default is to build universal binaries 
+(natively supporting both the Intel and ARM processors). The ``-DCMAKE_OSX_ARCHITECTURES`` option 
+may be set to just arm64 or x86_64 to override the default value, which is ``arm64;x86_64``.
+
+When doing a universal build, note that the OCIO dependencies must be built as universal libraries 
+too. If you are running in OCIO_INSTALL_EXT_PACKAGES=MISSING or NONE mode, your build will fail if 
+any of your installed libraries are not universal. The easiest way to address this is to set 
+OCIO_INSTALL_EXT_PACKAGES=ALL in order to let OCIO build everything. Alternatively, you may set 
+CMAKE_OSX_ARCHITECTURES to just the platform you are targeting.
```

Comment:
Trying to look at a few other ASWF projects (Imath, OpenEXR, MaterialX, OIIO), I don't see any using universal build as the default setting I think (they do not seem to touch `CMAKE_OSX_ARCHITECTURES`). I wonder if there are other projects doing this?

Reply:
I agree with your concerns. It might not be the best default if we consider an older MacBook that won't be able to build for ARM. 

However, I assume that the most recent MacBook (intel or arm) can build the universal build?

Reply:
I will be removing the universal build by default.

Reply:
Yeah I think my concerns are mostly that the builds will most likely not work anymore for most users unless using `OCIO_INSTALL_EXT_PACKAGES=ALL` and also will take more time, resources and size than needed for people that are not packaging OCIO but only doing local builds. I guess also another thing would be to align with other projects using CMake to have predictable behaviour, not sure if there is a consensus here, I only sampled 4 projets.

Reply:
I think it makes sense to keep the current architecture as the default build. I'll remove the code from the CMakeLists.txt